example/serial-fpp/test_fixtured_module_fpp.F90

```diff
@@ -90,6 +92,7 @@ subroutine random_test_suite_set_up(this)
 
     ! Note: factorial(n) with n > 13 overflows with 32 bit integers
     nn = int(13 * rand) + 1
+    this%rn = nn
```

Comment:
This should be removed, as we do not want to use variables within the type instance in this example.